packages/edit-site/src/components/sidebar-edit-mode/global-styles-sidebar.js

```diff
@@ -141,7 +141,7 @@ export default function GlobalStylesSidebar() {
 					gap={ 1 }
 				>
 					<FlexBlock style={ { minWidth: 'min-content' } }>
-						<h2 className="interface-complementary-area-header__title">
+						<h2 className="edit-site-global-styles-sidebar__header-title">
```

Comment:
Should this class actually change? As part of the editor unification work, it's my impression we are moving away from the separate `edit-site-`, `edit-post-` prefixes, instead of unified classes. Just making sure this was intentional. CC: @youknowriad

Reply:
I undid the change, to continue using `.edit-site-global-styles-sidebar__header-title` for the time being.

Reply:
We're just following our CSS guidelines while unifying. If we move stuff to the "editor" package, the class will have to change the prefix but otherwise it doesn't.